Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -33,15 +33,29 @@ class HTTPClient {
     private let signing: SigningType
     private let diagnosticsTracker: DiagnosticsTrackerType?
     private let dateProvider: DateProvider
+    private let retriableStatusCodes: Set<HTTPStatusCode>
+    private let operationDispatcher: OperationDispatcher
+
+    #if DEBUG
+    private var pathRequestCounts: [String: Int] = [:]
```

Comment:
Hmm I think we're not resetting these accross multiple requests? Probably ok for now since it's only used on tests for now though.